### ISSUE TITLE
After git pull to 385b2fbcec, mouse right click on outline show errors

### ISSUE THREAD

**Issue by wangzhaohe** [leo-editor/leo-editor#2806]
After git pull to 385b2fbcec, mouse right click on outline show errors.

```
Exception executing right-click handler
Traceback (most recent call last):
  File "/Users/swot/app/leo-editor/leo/plugins/qt_gui.py", line 1220, in onContextMenu
    handler(c, p, menu)
  File "/Users/swot/app/leo-editor/leo/plugins/todo.py", line 122, in popup_entry
    c.cleo.addPopupMenu(c, p, menu)
  File "/Users/swot/app/leo-editor/leo/plugins/todo.py", line 481, in addPopupMenu
    icon = self.menuicon(i)
  File "/Users/swot/app/leo-editor/leo/plugins/todo.py", line 525, in menuicon
    if pri_s not in self.menuicons:
UnboundLocalError: local variable 'pri_s' referenced before assignment
```

Leo Log Window
Leo 6.7-devel, devel branch, build 385b2fbcec
2022-09-04 09:05:06 +0800
Python 3.9.5, PyQt version 5.15.2
darwin

**Comment by edreamleo** [comment]
@wangzhaohe Thanks for this report!  I discovered this myself yesterday. The fix is in PR #2805, which was supposed to be about something else :-)

**Comment by edreamleo** [comment]
@wangzhaohe I just merged PR #2805 into devel. mypy isn't happy, but that's a nit :-)
